src/NHibernate/SqlCommand/SqlStringHelper.cs

```diff
@@ -43,6 +43,7 @@ public static SqlString[] Add(SqlString[] x, string sep, SqlString[] y)
 		}
 
 
+		//Consider using CriterionUtil.GetColumn... methods instead
```

Comment:
Consider removing the above undue blank line by the way.
And also, for better discoverability of the advice:
```suggestion
		/// <summary>
		/// Remove the <c>as someColumnAlias<c/> clause from a <c>SqlString</c> representing a column expression.
		/// Consider using <c>CriterionUtil.GetColumn...</c> methods instead.
		/// </summary>
```

Reply:
Done